.buildkite/pipeline-raw.yml

```diff
@@ -170,7 +170,7 @@ steps:
    - buildkite-agent artifact download "docker/*" arm-binaries --step "build-arm-via-docker"
    - ls binaries
    - ./scripts/autorelease.sh "$BUILDKITE_MESSAGE"
-   branches: master
+   #branches: master
```

Comment:
Don't forget to drop this change:)

Reply:
_spoiler: he did indeed forget to drop this change_